OpenRA.Mods.D2k/Activities/SwallowActor.cs

```diff
@@ -91,11 +95,13 @@ bool AttackTargets(Actor self, IEnumerable<Actor> targets)
 				}
 			});
 
+			var barrel = armament.CheckFire(self, facing, target);
```

Comment:
The `WormJaw` defines a `SpreadDamage` warhead that will now be applied.  We'll want to remove that.